raw/dune

```diff
@@ -0,0 +1,196 @@
+(library
+ (name raw)
+ (public_name hacl-star-raw)
+ (libraries ctypes ctypes.stubs lib_stubs lib_bindings)
+ (modules)
+ (wrapped false))
+
+(rule
+ (deps configure
+       (source_tree ../kremlin))
+ (targets
+  config.h
+  Makefile.config)
+ (action
+  (no-infer
+    (run ./configure))))
+
+(library
+  (name lib_evercrypt_so)
+  (public_name hacl-star-raw.lib-evercrypt-so)
+  (modules)
+  (foreign_archives evercrypt))
+
+(rule
+  (deps config.h Makefile Makefile.config (source_tree .))
+  (targets libevercrypt.a dllevercrypt.so)
+  (action
+    (no-infer
+      (progn
+        (run make libevercrypt.so libevercrypt.a)
```

Comment:
I think you're supposed to use `%{make}` so that it uses the right binary.

Reply:
Ah, indeed. That's convenient!